src/Symfony/Component/Security/Core/Authorization/Voter/RoleVoter.php

```diff
@@ -57,6 +58,10 @@ public function vote(TokenInterface $token, $object, array $attributes)
         $roles = $this->extractRoles($token);
 
         foreach ($attributes as $attribute) {
+            if ($attribute instanceof RoleInterface) {
+                $attribute = $attribute->getRole();
```

Comment:
We should not call `supportAttribute` if the return value of `getRole()` is `null` as it means that there is no good representation of the role as a string.